### DISCUSSION TITLE
Cannot install copilot

### DISCUSSION THREAD

**Discussion by aurkaxi** [community/community#32588] [Copilot Conversations] [answered]
on the marketplace there is nothing called copilot. So i downloaded that from [official web](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot&ssr=false#overview). and installed it. 
now it says copilot requires vscode v1.51+. i have 1.71 installed.
![2022-09-11_01-00](https://user-images.githubusercontent.com/72933395/189498106-00ea3992-fb34-432b-8e1d-f297e21df786.png)

**Comment by davecheney** [comment, marked as answer]
@ashroyxi I'm sorry OSS Code is not supported at the moment. We know it's something people would like to use, but it turns out while OSS Code is built from the same source code as VSCode, they deviate in non trivial ways. Please use VSCode.
